src/MasterSite.php

```diff
@@ -325,6 +325,52 @@ public function clean_post_cache( $post_id, $post_after, $post_before ) {
 		if ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) {
 			return;
 		}
+
+		// Flush TAB posts cache, if the selected act page status change.
```

Comment:
Maybe it's easier if we put this in a separate function. That way you can invert the 2 if-statements inside to return from that function, so we don't need to wrap the statements inside of them.

Reply:
So would be something like
```
private function check_act_page_change ($post_id) {
  if ( [[not the act page]] ) {
    // nothing to do
    return;
  }
  if ( [[status didn't change]] ) {
    // nothing to do
    return;
  }
}
```